clients/rust/Cargo.toml

```diff
@@ -23,7 +23,7 @@ num-traits = "^0.2"
 serde = { version = "^1.0", features = ["derive"], optional = true }
 serde_with = { version = "^3.0", optional = true }
 solana-program = "~1.18.11"
-thiserror = "^1.0"
+thiserror = "1.0.63"
```

Comment:
Was specifying the minor version required? In other words, will it not work with 1.0.62?

Reply:
Specifying minor version required to avoid dependency conflicts in packages that import mpl-bubblegum. Previously (with "^1.0" version) thiserror caused conflicts with other dependencies. However, as far as we do not use package = "=version_number" syntax, the compiler can choose another minor version than 63 as long as it do not cause any conflicts